docs/user-manual/civiform-admin-guide/using-enumerator-questions-and-screens-in-a-program.md

```diff
@@ -89,3 +88,113 @@ Any question type can be a repeated question, including the enumerator type.
+> | **Name**  |    **Birth Date**  | **Relationship** | **Employment Status**  |
+> |-----------|--------------------|------------------|------------------------|
+> |           |                    |                  |                        |
+> |           |                    |                  |                        |
+> |           |                    |                  |                        |
+
+   Example valid values: \
+   Relationship: "Head of Household", "Spouse", "Child", "Parent"  
+   Employment Status: "Employed", "Unemployed", "Retired", "Student"
+
+---
+
+#### Steps to Collect Household Information
+
+##### Step 1: Create an Enumerator Question
+1. Go to the **Questions** tab and click "Create new question".
+2. Select **"Enumerator"** as the question type.
+3. Fill in the details:
+> | **Field** |    **Value**  | 
+> |-----------|--------------------|
+> |Question text   | `List all members of your household`                |
+> |Question help text           |     Add any helpful text for the applicant               |
+> |Administrative identifier           | `household_members`                   |
+> |Question enumerator           |Set to `does not repeat` since this is the top-level enumerator |
+> |Question settings - Repeated entity type| <p> `Household member` </p>This is the logical description of the repeated entity or the name that will appear when applicants add entities to or remove entities from the enumerator question. For example, setting this field to 'household member' would show: “add household member”, “remove household member”, “name of household member”, etc, on subsequent screens. |
+
+4. Click **Create**.
+
+---
+
+##### Step 2: Create Repeated Questions
+Now, create the repeated questions for each piece of information you want to collect about each entity (in this case, `household member`). For this example, we'll create repeated questions for:
+
+**Name**:
+1. Create a new question of type **"Name"**.
+2. Fill in the details:
+   - **Question text**: `What is $this's full legal name?`.  (The **$this** token refers to the household member listed by the applicant.)
+
+   - **Administrative identifier**: `household_member_name`
+   - **Question enumerator**: `household_members` (This is the Administrative identifier of the Enumerator Question created in the previous step.) 
+
+3. Click **Create**.
+
+**Birth Date**:
+1. Create a new question of type **"Date"**.
+2. Fill in the details:
+   - **Question text**: `What is $this's date of birth?`
+   - **Administrative identifier**: `household_member_birthdate`
+   - **Question enumerator**: `household_members`
+3. Click **Create**.
+
+   Tip:  By asking for the date of birth and not age directly,  we can calculate the age, and the benefit is  that the value of age is only valid at a specific point of time and becomes stale as time goes on, whereas a person's date of birth stays valid.  a slightly shorter application for the applicants. 
```

Comment:
**nitpick:** a slightly shorter application for the applicants. -> This also makes for a slightly shorter application for the applicants as opposed to asking both questions.